| Pr     | 1.0000 | 1.2000 | 1.5000 | 2.0000 | 3.0000 | 5.0000 | 7.0000 | 10.000 |
|--------|--------|--------|--------|--------|--------|--------|--------|--------|
| Tr     |        |        |        |        |        |        |        |        |
| 0.30   | -11.062| -11.055| -11.044| -11.027| -10.992| -10.935| -10.872| -10.781|
| 0.35   | -10.640| -10.637| -10.632| -10.624| -10.609| -10.581| -10.554| -10.529|
| 0.40   | -10.121| -10.121| -10.121| -10.122| -10.123| -10.128| -10.135| -10.150|
| 0.45   | -9.525 | -9.527 | -9.531 | -9.537 | -9.549 | -9.576 | -9.611 | -9.663 |
| 0.50   | -8.888 | -8.892 | -8.899 | -8.909 | -8.932 | -8.978 | -9.030 | -9.111 |
| 0.55   | -8.243 | -8.243 | -8.252 | -8.267 | -8.298 | -8.360 | -8.245 | -8.531 |
| 0.60   | -7.596 | -7.603 | -7.611 | -7.632 | -7.669 | -7.745 | -7.824 | -7.950 |
| 0.65   | -6.980 | -6.987 | -6.997 | -7.017 | -7.059 | -7.147 | -7.239 | -7.381 |
| 0.70   | -6.338 | -6.395 | -6.407 | -6.429 | -6.475 | -6.574 | -6.677 | -6.837 |
| 0.75   | -5.824 | -5.832 | -5.845 | -5.868 | -5.918 | -6.027 | -6.142 | -6.318 |
| 0.80   | -5.285 | -5.293 | -5.306 | -5.330 | -5.385 | -5.506 | -5.632 | -5.824 |
| 0.85   | -4.763 | -4.771 | -4.784 | -4.810 | -4.872 | -5.000 | -5.149 | -5.358 |
| 0.90   | -4.249 | -4.255 | -4.268 | -4.298 | -4.371 | -4.530 | -4.688 | -4.916 |
| 0.93   | -3.934 | -3.937 | -3.951 | -3.987 | -4.073 | -4.251 | -4.422 | -4.662 |
| 0.95   | -3.712 | -3.713 | -3.730 | -3.773 | -3.873 | -4.068 | -4.248 | -4.497 |
| 0.97   | -3.470 | -3.467 | -3.492 | -3.551 | -3.670 | -3.885 | -4.077 | -4.336 |
| 0.98   | -3.332 | -3.327 | -3.363 | -3.434 | -3.568 | -3.795 | -3.992 | -4.257 |
| 0.99   | -3.164 | -3.164 | -3.223 | -3.313 | -3.464 | -3.705 | -3.909 | -4.178 |
| 1.00   | -2.471 | -2.952 | -3.065 | -3.186 | -3.358 | -3.615 | -3.825 | -4.100 |
| 1.01   | -1.375 | -2.595 | -2.880 | -3.051 | -3.251 | -3.525 | -3.742 | -4.023 |
| 1.02   | -1.180 | -1.723 | -2.650 | -2.906 | -3.142 | -3.435 | -3.661 | -3.947 |
| 1.05   | -0.877 | -0.878 | -1.496 | -2.381 | -2.800 | -3.167 | -3.418 | -3.722 |
| 1.10   | -0.617 | -0.673 | -0.617 | -1.261 | -2.167 | -2.720 | -3.023 | -3.362 |
| 1.15   | -0.459 | -0.503 | -0.487 | -0.604 | -1.497 | -2.275 | -2.641 | -3.019 |
| 1.20   | -0.349 | -0.381 | -0.381 | -0.361 | -0.934 | -1.840 | -2.273 | -2.692 |
| 1.30   | -0.203 | -0.218 | -0.218 | -0.178 | -0.300 | -1.066 | -1.592 | -2.086 |
| 1.40   | -0.111 | -0.115 | -0.128 | -0.070 | -0.044 | -0.504 | -1.012 | -1.547 |
| 1.50   | -0.049 | -0.046 | -0.032 | 0.008  | 0.078  | -0.142 | -0.556 | -1.080 |
| 1.60   | -0.005 | 0.004  | 0.023  | 0.065  | 0.151  | 0.082  | -0.217 | -0.689 |
| 1.70   | 0.027  | 0.040  | 0.063  | 0.109  | 0.202  | 0.223  | 0.028  | -0.369 |
| 1.80   | 0.051  | 0.067  | 0.094  | 0.143  | 0.241  | 0.317  | 0.203  | -0.112 |
| 1.90   | 0.070  | 0.088  | 0.117  | 0.169  | 0.271  | 0.381  | 0.330  | 0.092  |
| 2.00   | 0.085  | 0.105  | 0.136  | 0.190  | 0.295  | 0.428  | 0.424  | 0.255  |
| 2.20   | 0.106  | 0.128  | 0.163  | 0.221  | 0.331  | 0.493  | 0.551  | 0.489  |
| 2.40   | 0.120  | 0.144  | 0.181  | 0.242  | 0.356  | 0.535  | 0.631  | 0.645  |
| 2.60   | 0.130  | 0.156  | 0.194  | 0.257  | 0.376  | 0.567  | 0.687  | 0.754  |
| 2.80   | 0.137  | 0.164  | 0.204  | 0.269  | 0.391  | 0.591  | 0.729  | 0.836  |
| 3.00   | 0.142  | 0.170  | 0.211  | 0.278  | 0.403  | 0.611  | 0.763  | 0.899  |
| 3.50   | 0.152  | 0.181  | 0.224  | 0.294  | 0.425  | 0.650  | 0.827  | 1.015  |
| 4.00   | 0.158  | 0.188  | 0.233  | 0.306  | 0.442  | 0.680  | 0.874  | 1.097  |


In [11]:
import pandas as pd

data = {
    "Pr": [1.0000, 1.2000, 1.5000, 2.0000, 3.0000, 5.0000, 7.0000, 10.000],
    0.30: [-11.062, -11.055, -11.044, -11.027, -10.992, -10.935, -10.872, -10.781],
    0.35: [-10.640, -10.637, -10.632, -10.624, -10.609, -10.581, -10.554, -10.529],
    0.40: [-10.121, -10.121, -10.121, -10.122, -10.123, -10.128, -10.135, -10.150],
    0.45: [-9.525, -9.527, -9.531, -9.537, -9.549, -9.576, -9.611, -9.663],
    0.50: [-8.888, -8.892, -8.899, -8.909, -8.932, -8.978, -9.030, -9.111],
    0.55: [-8.243, -8.243, -8.252, -8.267, -8.298, -8.360, -8.245, -8.531],
    0.60: [-7.596, -7.603, -7.611, -7.632, -7.669, -7.745, -7.824, -7.950],
    0.65: [-6.980, -6.987, -6.997, -7.017, -7.059, -7.147, -7.239, -7.381],
    0.70: [-6.338, -6.395, -6.407, -6.429, -6.475, -6.574, -6.677, -6.837],
    0.75: [-5.824, -5.832, -5.845, -5.868, -5.918, -6.027, -6.142, -6.318],
    0.80: [-5.285, -5.293, -5.306, -5.330, -5.385, -5.506, -5.632, -5.824],
    0.85: [-4.763, -4.771, -4.784, -4.810, -4.872, -5.000, -5.149, -5.358],
    0.90: [-4.249, -4.255, -4.268, -4.298, -4.371, -4.530, -4.688, -4.916],
    0.93: [-3.934, -3.937, -3.951, -3.987, -4.073, -4.251, -4.422, -4.662],
    0.95: [-3.712, -3.713, -3.730, -3.773, -3.873, -4.068, -4.248, -4.497],
    0.97: [-3.470, -3.467, -3.492, -3.551, -3.670, -3.885, -4.077, -4.336],
    0.98: [-3.332, -3.327, -3.363, -3.434, -3.568, -3.795, -3.992, -4.257],
    0.99: [-3.164, -3.164, -3.223, -3.313, -3.464, -3.705, -3.909, -4.178],
    1.00: [-2.471, -1.375, -2.595, -2.952, -3.065, -3.186, -3.358, -3.615],
    1.01: [-3.825, -4.100, -2.880, -3.051, -3.251, -3.525, -3.742, -4.023],
    1.02: [-1.180, -1.723, -2.650, -2.906, -3.142, -3.435, -3.661, -3.947],
    1.05: [-0.877, -0.878, -1.496, -2.381, -2.800, -3.167, -3.418, -3.722],
    1.10: [-0.617, -0.673, -0.617, -1.261, -2.167, -2.720, -3.023, -3.362],
    1.15: [-0.459, -0.503, -0.487, -0.604, -1.497, -2.275, -2.641, -3.019],
    1.20: [-0.349, -0.381, -0.381, -0.361, -0.934, -1.840, -2.273, -2.692],
    1.30: [-0.203, -0.218, -0.218, -0.178, -0.300, -1.066, -1.592, -2.086],
    1.40: [-0.111, -0.115, -0.128, -0.070, -0.044, -0.504, -1.012, -1.547],
    1.50: [-0.049, -0.046, -0.032, 0.008, 0.078, -0.142, -0.556, -1.080],
    1.60: [-0.005, 0.004, 0.023, 0.065, 0.151, 0.082, -0.217, -0.689],
    1.70: [0.027, 0.040, 0.063, 0.109, 0.202, 0.223, 0.028, -0.369],
    1.80: [0.051, 0.067, 0.094, 0.143, 0.241, 0.317, 0.203, -0.112],
    1.90: [0.070, 0.088, 0.117, 0.169, 0.271, 0.381, 0.330, 0.092],
    2.00: [0.085, 0.105, 0.136, 0.190, 0.295, 0.428, 0.424, 0.255],
    2.20: [0.106, 0.128, 0.163, 0.221, 0.331, 0.493, 0.551, 0.489],
    2.40: [0.120, 0.144, 0.181, 0.242, 0.356, 0.535, 0.631, 0.645],
    2.60: [0.130, 0.156, 0.194, 0.257, 0.376, 0.567, 0.687, 0.754],
    2.80: [0.137, 0.164, 0.204, 0.269, 0.391, 0.591, 0.729, 0.836],
    3.00: [0.142, 0.170, 0.211, 0.278, 0.403, 0.611, 0.763, 0.899],
    3.50: [0.152, 0.181, 0.224, 0.294, 0.425, 0.650, 0.827, 1.015],
    4.00: [0.158, 0.188, 0.233, 0.306, 0.442, 0.680, 0.874, 1.097]
}

df = pd.DataFrame(data)
df.set_index("Pr", inplace=True)
df


0.30    0.35    0.40   0.45   0.50   0.55   0.60   0.65   0.70   0.75  \
Pr                                                                              
1.0  -11.062 -10.640 -10.121 -9.525 -8.888 -8.243 -7.596 -6.980 -6.338 -5.824   
1.2  -11.055 -10.637 -10.121 -9.527 -8.892 -8.243 -7.603 -6.987 -6.395 -5.832   
1.5  -11.044 -10.632 -10.121 -9.531 -8.899 -8.252 -7.611 -6.997 -6.407 -5.845   
2.0  -11.027 -10.624 -10.122 -9.537 -8.909 -8.267 -7.632 -7.017 -6.429 -5.868   
3.0  -10.992 -10.609 -10.123 -9.549 -8.932 -8.298 -7.669 -7.059 -6.475 -5.918   
5.0  -10.935 -10.581 -10.128 -9.576 -8.978 -8.360 -7.745 -7.147 -6.574 -6.027   
7.0  -10.872 -10.554 -10.135 -9.611 -9.030 -8.245 -7.824 -7.239 -6.677 -6.142   
10.0 -10.781 -10.529 -10.150 -9.663 -9.111 -8.531 -7.950 -7.381 -6.837 -6.318   

      ...   1.80   1.90   2.00   2.20   2.40   2.60   2.80   3.00   3.50  \
Pr    ...                                                                  
1.0   ...  0.051  0.070  0.085  0.106  0.120  0.130  0.137  0.142  0.152   
1.2   ...  0.067  0.088  0.105  0.128  0.144  0.156  0.164  0.170  0.181   
1.5   ...  0.094  0.117  0.136  0.163  0.181  0.194  0.204  0.211  0.224   
2.0   ...  0.143  0.169  0.190  0.221  0.242  0.257  0.269  0.278  0.294   
3.0   ...  0.241  0.271  0.295  0.331  0.356  0.376  0.391  0.403  0.425   
5.0   ...  0.317  0.381  0.428  0.493  0.535  0.567  0.591  0.611  0.650   
7.0   ...  0.203  0.330  0.424  0.551  0.631  0.687  0.729  0.763  0.827   
10.0  ... -0.112  0.092  0.255  0.489  0.645  0.754  0.836  0.899  1.015   

       4.00  
Pr           
1.0   0.158  
1.2   0.188  
1.5   0.233  
2.0   0.306  
3.0   0.442  
5.0   0.680  
7.0   0.874  
10.0  1.097  

[8 rows x 40 columns]

$$T_{media} = \left(\frac{0.213 + 0.216 + 0.208 + 0.214}{4}\right) \approx 0.2128$$

$$D_{T} = (T_{i} - T_{media}) = 0.0034$$

$$C_{x} = \left( \frac{-log T}{ε b} \right) = \frac{-log (0.2128)}{3312} = 2.029 * 10^{-4}M$$

Donde:

$$S_{y} = (0.434) \frac{S_{T}}{T} = 0.434 * \left( \frac{0.0034}{0.2128} \right) \approx 0.00693$$

$$C_{x} = \frac{-log T}{ε b} = \frac{0.672 \plusmn 0.00693}{3312 \plusmn 12}$$

$$\frac{S_{C_{x}}}{C_{x}} = \sqrt{ \left(\frac{0.00693}{0.672} \right) ^2 + \left( \frac{12}{3312} \right) ^2} \approx 0.0109$$

$$S_{C_{x}} = (0.0109)*(2.029*10^{-4}) \approx 2.22*10^{-6}$$

$$C_{V} = \left( \frac{2.22*10^{-6}}{2.029*10^{-4}} \right) * 100\% \approx 1.1 \%$$


a.
$$y = \left( 4.17*10^{-4} \right) ^3 = 7.251 * 10^{-11}$$
$$\frac{S_{y}}{y} = 3 \left( \frac{0.003*10^{-4}}{4.17*10^{-4}} \right) \approx 0.0216$$
$$S_{y} = (0.02316)(7.251*10^{-11}) \approx 1.565*10^{-12}$$
$$y = 7.3*(\plusmn 0.2)*10^{-11}$$
$$C_{V} = \left( \frac{1.565*10^{-12}}{7.251*10^{-11}} \right) * 100\% \approx 2.2\%$$

b.
$$y = 1.3090 *(\plusmn 0.0002)$$
$$C_{V} = 0.02\%$$